doc/contributing/maintaining-postject.md

```diff
@@ -12,12 +12,16 @@ Check that Node.js still builds and tests.
 
 ## Committing postject
 
-1. Add postject: `git add --all test/fixtures/postject-copy`.
+1. Add postject:
+   ```console
+   $ git add test/fixtures/postject-copy
+   $ git add deps/postject
+   ```
 2. Commit the changes: `git commit`.
 3. Add a message like:
 
    ```text
-   test: update postject to <version>
+   deps,test: update postject to <version>
```

Comment:
Why is this `test`? It should only be `deps`

Reply:
@anonrig the entirety of postject (including the resource injector, which is used purely in tests in node) is kept inside test/fixtures though. It's just the c header file that gets compiled into node and that's the only component that's getting copied to deps.